pyproject.toml

```diff
@@ -87,8 +87,8 @@ all = [
     "html5lib>=1.1",
     "bleach>=3.2.1",
     "pandas>=2.0",
-    "sortedcontainers>=1.5.7", # (older versions may work)
-    "pytz>=2016.10", # (older versions may work)
+    "sortedcontainers>=2.1.0",  # imposed by testing with hypothesis
```

Comment:
If you look up [the logs for the latest run of the oldest dependencies CI job on `main`](https://github.com/astropy/astropy/actions/runs/15630219178/job/44032597777#step:10:169) you will see that `sortedcontainers==2.1.0` is what that job already uses.